tests/sdk/flagsmith.test.ts

```diff
@@ -193,6 +193,15 @@ test('default flag handler used when timeout occurs', async () => {
     expect(flag.value).toBe(defaultFlag.value);
 })
 
+test('default request timeout in seconds', async () => {
```

Comment:
I think we should be more descriptive about what we're testing here? Something like: 

```suggestion
test('request timeout uses default if not provided', async () => {
```

Reply:
Thanks. Done